src/dispatchers/keyEventDispatcher.ts

```diff
@@ -0,0 +1,67 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Dispatcher {
+  export class KeyEvent {
```

Comment:
One can argue the same for most of Mouse...  I don't want to run into a situation where we have some `Dispatcher`s have `Event` appended to the end and some don't